devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -158,6 +226,31 @@ private handlePresenceEvent(present) {
     sendEvent(eventMap)
 }
 
+private handlePresenceEventVision(present) {
```

Comment:
As above, this method is not necessary. Please refer to my above comment regarding that.

Reply:
OK! I will delete this method.